Sources/Commands/SwiftPackageCollectionsTool.swift

```diff
@@ -0,0 +1,358 @@
+            let createdAt = DateFormatter().string(from: collection.createdAt)
+            let packages = collection.packages.map { "\($0.repository.url)" }.joined(separator: "\n")
+
+            print("""
+                Name: \(collection.name)
+                Source: \(collection.source.url)\(description)\(keywords)
+                Created At: \(createdAt)
+                Packages:
+                \(packages)
+            """)
+        }
+    }
+
+    // MARK: Search
+
+    enum SearchMethod: String, EnumerableFlag {
+        case keywords
+        case module
+    }
+
+    struct Search: ParsableCommand {
+        static var configuration = CommandConfiguration(abstract: "Search for packages by keywords or module names")
+
+        @OptionGroup
+        var jsonOptions: JSONOptions
+
+        @OptionGroup
+        var profileOptions: ProfileOptions
+
+        @Flag(help: "Pick the method for searching")
+        var searchMethod: SearchMethod
+
+        @Argument(help: "Search query")
+        var searchQuery: String?
+
+        mutating func run() throws {
+            guard let searchQuery = searchQuery else {
+                throw CollectionsError.missingArgument("searchQuery")
+            }
+
+            switch searchMethod {
+            case .keywords:
+                let results = try await { collections.findPackages(searchQuery, collections: nil, profile: profileOptions.usedProfile, callback: $0) }
+
+                results.items.forEach {
+                    print("\($0.package.repository.url): \($0.package.description ?? "")")
+                }
+
+            case .module:
+                let results = try await { collections.findTargets(searchQuery, searchType: .exactMatch, collections: nil, profile: profileOptions.usedProfile, callback: $0) }
+
+                results.items.forEach {
+                    $0.packages.forEach {
+                        print("\($0.repository.url): \($0.description ?? "")")
+                    }
+                }
+            }
+        }
+    }
+
+    // MARK: Packages
+
+    struct Describe: ParsableCommand {
+        static var configuration = CommandConfiguration(abstract: "Get metadata for a single package")
+
+        @OptionGroup
+        var jsonOptions: JSONOptions
+
+        @OptionGroup
+        var profileOptions: ProfileOptions
+
+        @Argument(help: "URL of the package to get information for")
+        var packageUrl: String?
+
+        @Option(name: .long, help: "Version of the package to get information for")
+        var version: String?
+
+        private func printVersion(_ version: PackageCollectionsModel.Package.Version?) -> String? {
+            guard let version = version else {
+                return nil
+            }
+
+            let modules = version.targets.compactMap { $0.moduleName }.joined(separator: ", ")
+            let platforms = optionalRow("Supported Platforms", version.verifiedPlatforms?.map { $0.name }.joined(separator: ", "))
```

Comment:
We might want to change the wording here "supported" -> "verified" to not confuse users with similar properties in `Package.swift` (they are different).